src/graphql/resolvers/user.ts

```diff
@@ -0,0 +1,57 @@
+import prismaClient from "../../client.js";
+import { User } from "../model.types/user.js";
+export const userResolver = {
+	id(parent: User) {
+		return parent.id;
+	},
+	username(parent: User) {
+		return parent.username;
+	},
+	passwordHash(parent: User) {
```

Comment:
Why expose the retrieval of a password hash to the client 😱?. This is a serious security risk